src/Kentico.Xperience.Algolia/Admin/AlgoliaModuleInstaller.cs

```diff
@@ -0,0 +1,325 @@
+﻿using CMS.DataEngine;
+using CMS.FormEngine;
+using CMS.Modules;
+
+namespace Kentico.Xperience.Algolia.Admin;
+
+internal class AlgoliaModuleInstaller
+{
+    private readonly IResourceInfoProvider resourceProvider;
+
+    public AlgoliaModuleInstaller(IResourceInfoProvider resourceProvider) => this.resourceProvider = resourceProvider;
+
+    public void Install()
+    {
+        var resource = resourceProvider.Get("CMS.Integration.Algolia")
+            // Handle v4.0.0 resource name manually until migrations are enabled
+            ?? resourceProvider.Get("Kentico.Xperience.Algolia")
```

Comment:
We don't need to handle this scenario here because we never released a package with a `ResourceInfo` named `Kentico.Xperience.Algolia`.

Reply:
deleted